data/json/items/archery.json

```diff
@@ -400,7 +400,51 @@
       "durability" : 6,
       "clip_size" : 1,
       "reload": 110,
-      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]]
+      "valid_mod_locations": [["underbarrel", 1],[ "sights", 1 ],[ "accessories", 2 ],["stabilizer", 1],["dampening", 1]],
+      "use_action": {
```

Comment:
Just personally, I feel like this might be better as two actions, one to tighten and one to loosen, and to change the compbow_high to loosen into the regular compbow.

Nothing necessarily wrong with cycling, but I feel like it's less intuitive.

Reply:
I agree completely, but I tried to do that when all this was just a mod, but having two activation actions on the same item just caused the game to crash. If there's a way to do this I'm not aware of it, but would be totally happy to.

Reply:
Well, I'm basing it on refillable lighters, which implement it like this:

```
    "use_action": [
      {
        "target": "ref_lighter_on",
        "msg": "You flick the lighter.",
        "active": true,
        "need_charges": 1,
        "need_charges_msg": "Nothing happens.",
        "menu_text": "Light up",
        "type": "transform"
      },
      {
        "type": "firestarter",
        "moves": 50
      }
    ]
```

I think it should be possible to just add the "tighten" and "loosen" as two different transform actions like this, right? Or does it not work with two of the same type of action, even if the target is different?

Reply:
I'd need a more experienced opinion on it, but I'm pretty sure those are two different actions, one to turn the lighter into an on version and one to start a fire.

Reply:
Well yeah, that's what I mean, two different actions. One to tighten, one to loosen. It's just that they would be both the same type of action, "transform", but it would have different targets. I was imagining something like:

```
"use_action": [
  {
    "menu_text": "Tighten Limbs",
    "type": "transform",
    "target": "compbow_high",
    "msg": "You tighten the limbs, increasing the strength of the bow."
  },
  {
    "menu_text": "Loosen Limbs",
    "type": "transform",
    "target": "compbow_low",
    "msg": "You loosen the limbs, decreasing the strength of the bow."
  }
]
```

But yeah, I'm also not exactly an experienced contributor here, so I may be missing something. It feels like it should work, though.

Reply:
That's exactly what I tried when it was a mod, and it made the game crash.

Reply:
Dang, that sucks. Could probably be considered a bug that you can't have two different transform actions in one item, but that's already not part of the scope of this PR, so nevermind then.

Reply:
It's possible I did it wrong originally or there's some special trick to it, but I don't know what it is.